helix-core/src/main/java/org/apache/helix/controller/stages/CurrentStateComputationStage.java

```diff
@@ -364,10 +364,6 @@ void handleResourceCapacityCalculation(ClusterEvent event, ResourceControllerDat
         .filter(entry -> WagedValidationUtil.isWagedEnabled(cache.getIdealState(entry.getKey())))
         .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
 
-    if (wagedEnabledResourceMap.isEmpty()) {
```

Comment:
Merge this map with map of resource configs? That would get resources with config but no EV/IS yet